### DISCUSSION TITLE
[Feature] Add Immich to Android Cloud Media Apps for Photos/Media

### DISCUSSION THREAD

**Discussion by ResponsiblyUnranked** [immich-app/immich#9433] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I recently noticed that Android 14 has the ability to choose different default 'Cloud Media App(s)' for media and photos. It would be nice to see Immich in this list as the default media provider.

On a Pixel 8 Pro running Android 14.

![screenshot](https://github.com/immich-app/immich/assets/51296820/32b64253-263b-4a2e-8983-abd713492f9d)

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#7101
